scripts/us_nces/README_download.md

```diff
@@ -0,0 +1,55 @@
+# Downloading input files for US: National Center for Education Statistics
+
```

Comment:
Could you please keep only one README.md file and mention download steps there which you wrote in "README_Download.md" file.